hypershift-operator/controllers/nodepool/kubevirt/kubevirt.go

```diff
@@ -235,7 +235,8 @@ func virtualMachineTemplateBase(nodePool *hyperv1.NodePool, bootImage BootImage)
 	dataVolume := kubevirtv1.DataVolumeTemplateSpec{
 		TypeMeta: metav1.TypeMeta{},
 		ObjectMeta: metav1.ObjectMeta{
-			Name: rootVolumeName,
+			Name:   rootVolumeName,
+			Labels: map[string]string{hyperv1.NodePoolNameLabel: nodePool.Name},
```

Comment:
I agree with the need in a dedicated label. but I'm not sure about boolean label. I think it will be better to have cluster or even node specific label.